apiv6/localhost_apiserver/main.go

```diff
@@ -10,20 +10,29 @@
 package main
 
 import (
+	"flag"
 	"log"
 	"net/http"
 
 	cloudexportstub "github.com/kentik/community_sdk_golang/apiv6/localhost_apiserver/go"
 )
 
 func main() {
-	log.Printf("Server started")
+	port, storageFilePath := readPortStorage()
 
-	repo := cloudexportstub.NewCloudExportRepo("CloudExportStorage.json")
+	repo := cloudexportstub.NewCloudExportRepo(storageFilePath)
 	CloudExportAdminServiceApiService := cloudexportstub.NewCloudExportAdminServiceApiService(repo)
 	CloudExportAdminServiceApiController := cloudexportstub.NewCloudExportAdminServiceApiController(CloudExportAdminServiceApiService)
 
 	router := cloudexportstub.NewRouter(CloudExportAdminServiceApiController)
 
-	log.Fatal(http.ListenAndServe(":8080", router))
+	log.Printf("Server started, port %s, storage %s", port, storageFilePath)
+	log.Fatal(http.ListenAndServe(port, router))
+}
+
+func readPortStorage() (port string, storage string) {
+	flag.StringVar(&port, "port", ":8080", "Port number for the server to listen to")
```

Comment:
```suggestion
	flag.StringVar(&address, "address", ":8080", "TCP address for the server to listen to")
```
If only port is passed (`8080`), the server will fail to start, I believe.

Reply:
updated port -> address

Reply:
Yes, exactly, ":8080" is address and "8080" is port, this is what I meant.